workers/unity/Packages/com.improbable.gdk.core/Config/ConnectionConfig.cs

```diff
@@ -29,6 +67,14 @@ public ConnectionParameters CreateConnectionParameters()
             return connectionParameters;
         }
 
+        /// <summary>
+        ///     Basic validation function which checks that a string value is non-null or empty.
```

Comment:
Is "Basic" appropriate here? I feel it's redundant

Reply:
Maybe - this function grates on me a lot though